ratbagd/ratbagd.h

```diff
@@ -169,6 +169,12 @@ struct ratbagd_device *ratbagd_device_next(struct ratbagd_device *device);
 	     (_device);					\
 	     (_device) = ratbagd_device_next(_device))
 
+#define RATBAGD_DEVICE_FOREACH_SAFE(_device, _safe, _ctx)	\
+	for ((_device) = ratbagd_device_first(_ctx),		\
```

Comment:
Parentheses on lvalues have weird semantics. I'd drop them. I even wonder why it works here.